code/index.html

```diff
@@ -1,22 +1,132 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+
+  <!-- Header -->
+
+  <header class="header">
+
+    <img src="images/goodnewspaper.JPG" alt="Newspaper title">
```

Comment:
Isn't this the logotype?
```suggestion
    <img src="images/goodnewspaper.JPG" alt="Logotype">
```